pytorch_lightning/core/lightning.py

```diff
@@ -399,7 +399,7 @@ def training_step(self, *args, **kwargs):
                 :paramref:`~pytorch_lightning.trainer.trainer.Trainer.truncated_bptt_steps` > 0.
 
         Return:
-            roch.Tensor or a dictionary with anything you want (must include the keyword 'loss')
+            :class:`~torch.Tensor` or a dictionary with anything you want (must include the keyword 'loss')
```

Comment:
```suggestion
            Either a :class:`~torch.Tensor` or a dictionary with anything
                you want (must include the keyword 'loss') or None
```
I believe now you can skip training_step by returning None.